### PR TITLE
Automate_upload_file_to_file_repo

### PR DESCRIPTION
**Pull request by vikask18**
PR for merge to master branch.
- Updated file_to_file_upload test case.
- Removed duplicate reads - as per @oshtaier comment in previous PR #6899.
- Test run is successful on both 6.5.z / 6.6 version of Sat 6.
- Added check for file by name in repository - @oshtaier
- Minor change in Docstring - @vijay8451

### CODE REVIEW

File: tests/foreman/cli/test_repository.py [hunk 1]
```diff
@@ -2686,8 +2685,29 @@ def test_positive_upload_file_to_file_repo(self):
 
         :expectedresults: uploaded file is available under File Repository
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
```
Comment (vijay8451) [inline tests/foreman/cli/test_repository.py]:
```suggestion
        :CaseAutomation: Automated
```

File: tests/foreman/cli/test_repository.py [hunk 2]
```diff
@@ -2686,8 +2685,29 @@ def test_positive_upload_file_to_file_repo(self):
...
+            'organization': new_repo['organization'],
+            'path': "/tmp/{0}".format(RPM_TO_UPLOAD),
+            'product-id': new_repo['product']['id'],
+        })
+        self.assertIn(
+            "Successfully uploaded file '{0}'".format(RPM_TO_UPLOAD),
+            result[0]['message'],
+        )
+        repo = Repository.info({'id': new_repo['id']})
+        self.assertEqual(repo['content-counts']['files'], '1')
```
Comment (oshtaier) [inline tests/foreman/cli/test_repository.py]: can you also check that file is one we expect? like filename is same as we uploaded?

Comment (vikask18) [reply tests/foreman/cli/test_repository.py]: Added the check for filename.

File: tests/foreman/cli/test_repository.py (hunk 1 above)
Comment (vikask18) [reply tests/foreman/cli/test_repository.py]: Done.

**Comment by vikask18** [comment]
Test Result:
----------------
(venvrobottelo65) [vkaushik@vkaushik robottelo]$ pytest tests/foreman/cli/test_repository.py -k test_positive_upload_file_to_file_repo -v
========================== test session starts ===============================
platform linux -- Python 3.6.6, pytest-4.0.2, py-1.8.0, pluggy-0.11.0 -- /home/vkaushik/RobotteloFramework/robottelo/venvrobottelo65/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/vkaushik/RobotteloFramework/robottelo, inifile:
plugins: services-1.3.1, mock-1.10.0, forked-1.0.2
collecting ... 2019-05-13 15:04:19 - conftest - DEBUG - BZ deselect is disabled in settings

collected 94 items / 93 deselected                                                                                                                                                        

tests/foreman/cli/test_repository.py::FileRepositoryTestCase::test_positive_upload_file_to_file_repo PASSED                                                                         [100%]

=================== 1 passed, 93 deselected in 25.52 seconds ======================

**Pull request merged**